src/System.Runtime/tests/System/StringGetHashCodeTests.cs

```diff
@@ -0,0 +1,60 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using Xunit;
+
+namespace System.Tests
+{
+    public class StringGetHashCodeTests : RemoteExecutorTestBase
+    {
+        [Fact]
+        public void GetHashCode_UseSameStringInTwoProcesses_ReturnsDifferentHashCodes()
+        {
+            EnsureHashCodeDifferentForChildString("abc".GetHashCode().ToString());
+        }
+
+        private static void EnsureHashCodeDifferentForChildString(string origHashCode, string timesInvoked = "0")
+        {
+            RemoteInvoke((parentHash, stimes) =>
+            {
+                int times;
+                int.TryParse(stimes, out times);
+
+                string childHash = "abc".GetHashCode().ToString();
+                if (parentHash.Equals(childHash) && times < 3)
+                {
+                    // 1 in 4 billion chance the child and parent hashcode are the same. To further reduce change of collision we retry up to 3 times
+                    EnsureHashCodeDifferentForChildString(parentHash, (times + 1).ToString());
+                }
+                else
+                {
+                    Assert.NotEqual(parentHash, childHash.ToString());
+                }
+                return SuccessExitCode;
+            }, origHashCode.ToString(), (timesInvoked + 1).ToString()).Dispose();
+        }
+
+        [Fact]
+        public void GetHashCode_UseDictionary_SetThreshold_UsesNonDefaultComparer()
+        {
+            var dict = new Dictionary<string, int>(EqualityComparer<string>.Default);
+            Assert.Equal("System.Collections.Generic.NonRandomizedStringEqualityComparer", dict.Comparer.GetType().ToString());
+            
+            foreach (string s in StringsMatchingNonRandomizedHashCode.Data)
+            {
+                dict.Add(s, 0);
+                if (dict.Count > 100)
```

Comment:
You could remove this and just change to `.. in StringsMatchingNonRandomizedHashCode.Data.Take(101)`

Reply:
or not bother at all because you just have 101 strings (well 102) just put them all in